sdk/communication/azure-communication-chat/src/main/java/com/azure/android/communication/chat/ChatAsyncClient.java

```diff
@@ -67,6 +71,8 @@ public ChatThreadAsyncClient getChatThreadClient(String chatThreadId) {
      * Creates a chat thread.
      *
      * @param options Options for creating a chat thread.
+     * @throws ChatErrorResponseException thrown if the request is rejected by server.
+     * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
```

Comment:
I don't think we need to add `RequestException` to the JavaDoc. What do you think @srnagar, @JonathanGiles?

Reply:
I notice in JAVA SDK, RequestException is added. @vcolin7

Reply:
Oh my bad, I meant to say `RuntimeException`.

Reply:
I also meant to say in JAVA SDK RuntimeException is added. I made the same mistake...